lib/cp8_cli/github/pull_request.rb

```diff
@@ -35,7 +35,9 @@ def save
           to,
           from,
           title,
-          body
+          body,
+          draft: true,
+          accept: "application/vnd.github.shadow-cat-preview" # waiting for https://github.com/octokit/octokit.rb/pull/1114
```

Comment:
I think you can remove this now, seems like https://github.com/octokit/octokit.rb/pull/1114 was already merged.

Reply:
👍 will take a look in a subsequent PR.